src/introduction/009_FAQ.md

```diff
@@ -0,0 +1,19 @@
+---
+id: 009_FAQ
+title: FAQ  
+sidebar_label: FAQ  
+---
+
+1. **The Ansible seems to have finished successfully and the Personium Unit has been built. Next, I would like to create a Cell, but how do I do that?**  
+Let's try the Cell creation API, which is a Unit-level API, by referring to the [Cell creation tutorial](../unit-administrator/tutorial.md).  
+
+1. **What to do if I get an error message (PR401-AU-0008) when I try to perform operations (Cell operations or searching queries) on the Personium Unit using the UUT (Unit User Token)?**  
+Please make sure that the FQDN of the Unit certificate that you set up when you built your Personium Unit matches the FQDN of the Personium Unit you are currently accessing.  
```

Comment:
How about adding how to fix this problem if happens. In most of this situation, you can fix it by the following steps.
1. Recreate Unit certificate with correct CommonName.
2. Replace the Unit certificate
3. Restart Tomcat

Reply:
@tochi-y 
Should I add this [link](https://personium.io/docs/en/server-operator/Confirm_environment_settings/#unit-certificate) to help the reader?

Reply:
Thank you for the explanation. I will prepare the procedures for generating the Unit certificate.

Reply:
Updates on the additional procedures to recreate the Unit certificate.
![image](https://user-images.githubusercontent.com/29846750/112585585-6dba3080-8e3d-11eb-9c31-e8c31601f2c6.png)

Reply:
Japanese version also ready for review.
![image](https://user-images.githubusercontent.com/29846750/112590255-defde180-8e45-11eb-8c05-38246d3d8e79.png)